### DISCUSSION TITLE
Next Steps

### DISCUSSION THREAD

**Discussion by lunamorrow** [MDAnalysis/mdanalysis#4585] [GSoC Discussions]
I was just accepted as a GSoC contributor this morning which is super exciting! I want to get kicking early on. What is the best way to set up communication @hmacdope @xhgchen @cbouy that works for all of us? I would like to set up expectations, start planning specifics and refining my plan, and let you know of my anticipated availability (as the coding period falls over a short university break and most of a semester for me in Australia).

**Comment by hmacdope** [comment]
We have moved this conversation to discord.
